js/src/gtag-events/index.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { addAction } from '@wordpress/hooks';
+
+/**
+ * Internal dependencies
+ */
+import { namespace, actionPrefix } from './constants';
+import {
+	getProductObject,
+	retrievedVariation,
+	trackAddToCartEvent,
+} from './utils';
+
+/* global jQuery */
+
+addAction(
+	`${ actionPrefix }-cart-add-item`,
+	namespace,
+	( { product, quantity = 1 } ) => {
+		trackAddToCartEvent( product, quantity );
+	}
+);
```

Comment:
❓ 
I guess this is what Gutemberg block triggers? 

Would be cool to add just a line explaining so.